GSM/GSML3MMElements.cpp

```diff
@@ -97,10 +97,10 @@ void L3TimeZoneAndTime::parseV(const L3Frame& src, size_t& rp)
 	// Read it all into a localtime struct tm,
 	// then covert to Unix seconds.
 	struct tm fields;
-	// year
-	fields.tm_year = 2000 + src.readField(rp,4) + src.readField(rp,4)*10;
+	// years since 1900
+	fields.tm_year = 100 + src.readField(rp,4) + src.readField(rp,4)*10;
```

Comment:
Nope. I guess it's unclear. The GSM packet is listing time since 2000 and mktime wants it since 1900. So I add 100 to the GSM time.